recipes/pymartini/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "0.3.2" %}
+{% set build = 0 %}
+{% set name = "pymartini" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e586fb492459d6ab80f442929b622d1567d2dc77bcffcdf49ac5146567b22944
+
+build:
+  number: {{ build }}
```

Comment:
You can just explicitly put the build number here and not use the jinja templating. The regro bot will handle it correctly.

Reply:
:+1: